deployer/src/config_creator.py

```diff
@@ -74,23 +74,90 @@ def to_gitbook_config(config):
+                "selectors_key": "articles",
+                "tags": [
+                    "articles"
+                ]
+            }]
 
-    config["selectors"]["lvl0"] = OrderedDict((
-        ("selector", ".contents h1"),
-        ("default_value", "Documentation")
+        config["sitemap_urls"] = [
+            root + "sitemap.xml"]
+
+    config["selectors"] = OrderedDict((
+        ("homepage", OrderedDict((
+            ("lvl0", OrderedDict((
+                ("selector", ".contents h1"),
+                ("default_value", "pkgdown Home page")
+            ))),
+            ("lvl1", ".contents h2"),
+            ("lvl2", OrderedDict((
+                ("selector", ".contents h3"),
+                ("default_value", "Context")
+            ))),
+            ("lvl3", ".ref-arguments td, .ref-description"),
+            ("text", ".contents p, .contents li, .contents .pre")
+        ))),
+        ("reference", OrderedDict((
+            ("lvl0", ".contents h1"),
+            ("lvl1", OrderedDict((
+                ("selector", ".contents .name"),
+                ("default_value", "Argument")
+            ))),
+            ("lvl2", OrderedDict((
+                ("selector", ".ref-arguments th"),
+                ("default_value", "Description")
+            ))),
+            ("lvl3", ".ref-arguments td, .ref-description"),
+            ("text", ".contents p, .contents li")
+        ))),
+        ("articles", OrderedDict((
+            ("lvl0", ".contents h1"),
+            ("lvl1", ".contents .name"),
+            ("lvl2", OrderedDict((
+                ("selector", ".contents h2, .contents h3"),
+                ("default_value", "Context")
+            ))),
+            ("text",
+             ".contents p, .contents li, .tempate-article .contents .pre")
+        ))),
+        ("default", OrderedDict((
+            ("lvl1", ".contents h2"),
+            ("lvl2", ".contents h3, .contents th"),
+            ("lvl3", ".contents h4"),
+            ("lvl4", ".contents h5"),
+            ("text",
+             ".contents p, .contents li, .usage, .template-article .contents .pre")
+        )))
     ))
-    config["selectors"]["lvl1"] = ".contents h2"
-    config["selectors"]["lvl2"] = ".contents h3, .contents th"
-    config["selectors"]["lvl3"] = ".contents h4"
-    config["selectors"]["lvl4"] = ".contents h5"
-    config["selectors"][
-        "text"] = ".contents p, .contents li, .usage, .template-article .contents .pre"
     config["selectors_exclude"] = [".dont-index"]
-    config["custom_settings"] = {"separatorsToIndex": "_"}
-    config["scrap_start_urls"] = False
-    config["stop_urls"] = ["LICENSE-text.html"]
+    config["stop_urls"] = ["/reference/$",
+                           "/reference/index.html",
+                           "/articles/$",
+                           "/articles/index.html"]
+    config["custom_settings"] = {
+        "separatorsToIndex": "_",
+        "attributesToRetrieve": ["hierarchy",
```

Comment:
Why do we need to overwrite the `attributesToRetrieve`?

Reply:
It comes from the need from pkgdown that uses a different UI, they process record in another way:

![image](https://user-images.githubusercontent.com/32097720/49889355-90c59f80-fe41-11e8-8a45-649e2cfd8ddd.png)